spce/spce.go

```diff
@@ -0,0 +1,141 @@
+package spce
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type SPCE struct {
+	rw io.ReadWriter
+	channel string
+}
+
+// NewSPCE creates a new SPCE instance with the given ReadWriter and channel address in the format of two hexadezimal characters
+// The default channel address is "05"
```

Comment:
Is it possible to change the channel address on the device? If no, I we can "hard-code" the channel address.